Azure/azure-cli
# [AKS] `az aks upgrade`: Add forceupgrade settings to aks stable cli

**Related command**
add forceupgrade settings to aks stable cli 

**Description**
add forceupgrade settings to aks upgrade which is currently only available in aks update

**Testing Guide**
aks upgrade--resource-group={resource_group} --name={name} --enable-force-upgrade
aks upgrade--resource-group={resource_group} --name={name} --disable-force-upgrade
aks upgrade--resource-group={resource_group} --name={name} --upgrade-override-until 2020-01-01T22:30:17+00:00

This checklist is used to make sure that common guidelines for a pull request are followed.
